war/src/test/java/io/lumeer/engine/rest/UserGroupServiceIntegrationTest.java

```diff
@@ -0,0 +1,251 @@
+      assertThat(usersAndGroups.get(user2)).containsOnly(group1);
+   }
+
+   @Test
+   public void testGetGroups() throws Exception {
+      final Client client = ClientBuilder.newBuilder().build();
+
+      Response response = client.target(TARGET_URI).path(PATH_PREFIX).request(MediaType.APPLICATION_JSON).buildGet().invoke();
+      List<String> groups = response.readEntity(List.class);
+      assertThat(groups).isEmpty();
+
+      String group1 = "GetGroup1";
+      String group2 = "GetGroup2";
+      userGroupFacade.addGroups(organization, group1, group2);
+
+      response = client.target(TARGET_URI).path(PATH_PREFIX).request(MediaType.APPLICATION_JSON).buildGet().invoke();
+      groups = response.readEntity(List.class);
+
+      assertThat(groups).containsOnly(group1, group2);
+   }
+
+   @Test
+   public void testRemoveGroups() throws Exception {
+      final Client client = ClientBuilder.newBuilder().build();
+
+      String group1 = "RemoveGroup1";
+      String group2 = "RemoveGroup2";
+      userGroupFacade.addGroups(organization, group1, group2);
+
+      client.target(TARGET_URI).path(PATH_PREFIX).request(MediaType.APPLICATION_JSON)
+            .buildPut(Entity.entity(Arrays.asList(group1), MediaType.APPLICATION_JSON)).invoke();
+
+      List<String> groups = userGroupFacade.getGroups(organization);
+      assertThat(groups).containsOnly(group2);
+
+      client.target(TARGET_URI).path(PATH_PREFIX).request(MediaType.APPLICATION_JSON)
+            .buildPut(Entity.entity(Arrays.asList(group2), MediaType.APPLICATION_JSON)).invoke();
+
+      groups = userGroupFacade.getGroups(organization);
+      assertThat(groups).isEmpty();
+   }
+
+   @Test
+   public void testGetGroupsOfUser() throws Exception {
+      final Client client = ClientBuilder.newBuilder().build();
+
+      String user = "user";
+      String group1 = "GetGroupsOfUser1";
+      String group2 = "GetGroupsOfUser2";
+      userGroupFacade.addGroups(organization, group1, group2);
+      userGroupFacade.addUser(organization, user, group1, group2);
+
+      Response response = client.target(TARGET_URI)
+                                .path(PATH_PREFIX + "users/" + user + "/groups/")
+                                .request(MediaType.APPLICATION_JSON).buildGet().invoke();
+      List<String> groups = response.readEntity(List.class);
+
+      assertThat(groups).containsOnly(group1, group2);
+   }
+
+   // TODO
+   @Test
+   public void testGetUsersInGroup() throws Exception {
+      final Client client = ClientBuilder.newBuilder().build();
+
+      String group = "GetUsersInGroup";
+      String user1 = "user 1";
+      String user2 = "user 2";
+
+      userGroupFacade.addUser(organization, user1, group);
+      userGroupFacade.addUser(organization, user2, group);
+
+      // even command from facade does not work the way I expect, and I can't find out why... :-(
+      assertThat(userGroupFacade.getUsersInGroup(organization, group)).containsOnly(user1, user2);
+
```

Comment:
Oh I see. So this is so strange... try to repair `UserGroupFacadeIntegrationTest` to put organization code as parameter instead of id to check if there is everything okay.